style.css

```
@@ -1,4 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@1,9..40,500&family=Poppins:wght@100;400&display=swap');
+
+
 * {
     margin: 0;
+    padding: 0;
     box-sizing: border-box;
+
+}
+
+.navigation {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-left: 177px;
+    margin-right: 230px;
+    margin-top: 41px;
```

Você pode abreviar o uso das margens assim.
```suggestion
    margin: 41px 230px 177px;
```